index.js

```diff
@@ -183,14 +183,15 @@ CleanWebpackPlugin.prototype.apply = function(compiler) {
   if (compiler === undefined) {
     return clean.call(_this);
   } else {
+    var pluginName = "clean-webpack-plugin";
```

Comment:
Move this variable declaration to the top of the file please, this is something really unique and probably is better to extract it from the ejection.